scripts/bump-version.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -eux
+
+if [ "$(uname -s)" != "Linux" ]; then
+    echo "Please use the GitHub Action."
+    exit 1
+fi
+
+SCRIPT_DIR="$( dirname "$0" )"
+cd $SCRIPT_DIR/..
+
+OLD_VERSION="${1}"
+NEW_VERSION="${2}"
+
+echo "Current version: $OLD_VERSION"
+echo "Bumping version: $NEW_VERSION"
+
+function replace() {
+    ! grep "$2" $3
+    perl -i -pe "s/$1/$2/g" $3
+    grep "$2" $3  # verify that replacement was successful
+}
+
+replace "version=\"[0-9.]+\"" "version=\"$NEW_VERSION\"" ./setup.py
```

Comment:
Version string may include dashes and letters too sometimes?

Reply:
Relaxed in 64c0104cd6f417acebbfd9819098355142f39b86. 👍